src/Raven.Studio/typescript/components/pages/resources/databases/partials/create/formBackup/CreateDatabaseFromBackup.tsx

```diff
@@ -0,0 +1,249 @@
+                        <Steps
+                            current={currentStep}
+                            steps={activeSteps.map(createDatabaseUtils.mapToStepItem)}
+                            onClick={(step) => goToStepWithValidation(step, validateToTargetStep(step - 1))}
+                            className="flex-grow me-4"
+                        ></Steps>
+                        <CloseButton onClick={closeModal} />
+                    </div>
+                    {stepViews[activeSteps[currentStep].id]}
+                </ModalBody>
+
+                <hr />
+                <ModalFooter>
+                    {isFirstStep ? (
+                        <Button
+                            type="button"
+                            onClick={changeCreateModeToRegular}
+                            className="rounded-pill"
+                            disabled={formState.isSubmitting}
+                        >
+                            <Icon icon="database" addon="star" /> Create new database
+                        </Button>
+                    ) : (
+                        <Button type="button" onClick={prevStep} className="rounded-pill">
+                            <Icon icon="arrow-thin-left" /> Back
+                        </Button>
+                    )}
+                    <FlexGrow />
+                    {isLastStep ? (
+                        <ButtonWithSpinner
+                            type="submit"
+                            color="success"
+                            className="rounded-pill"
+                            isSpinning={formState.isSubmitting}
+                            icon="rocket"
+                        >
+                            Finish
+                        </ButtonWithSpinner>
+                    ) : (
+                        <Button
+                            type="button"
+                            color="primary"
+                            className="rounded-pill"
+                            onClick={() => nextStepWithValidation(validateToTargetStep(currentStep))}
+                            disabled={asyncDatabaseNameValidation.loading}
+                        >
+                            Next <Icon icon="arrow-thin-right" margin="ms-1" />
+                        </Button>
+                    )}
+                </ModalFooter>
+            </Form>
+        </FormProvider>
+    );
+}
+
+type Step = CreateDatabaseStep<FormData>;
+
+function getActiveStepsList(
+    formValues: FormData,
+    errors: FieldErrors<FormData>,
+    isValidatingDatabaseName: boolean
+): Step[] {
+    const {
+        sourceStep: { sourceType, sourceData, isEncrypted },
+    } = formValues;
+
+    const isEncryptionActive =
+        sourceType && isEncrypted && !sourceData[sourceType].pointsWithTags[0].restorePoint?.isEncrypted;
+
+    const allSteps: Step[] = [
+        {
+            id: "basicInfoStep",
+            label: "Select backup",
```

Comment:
As I understand - 
the action done in this step **doesn't** "select backup" as this title suggests.
You select the target database type to create (regular or sharded),
for example, you can create a regular database from a sharded one.

so - maybe either one of these options:
`Select backup => Database type / Target database type / Target database / Database type to create`